src/sortStudents.ts

```diff
@@ -1,16 +1,64 @@
 
 export interface Student {
   // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
   // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'averageGrade',
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function getAverageGrade(grades: number[]): number {
+  return grades.reduce((a, b) => a + b, 0) / grades.length;
```

Comment:
Try to create more meaningfully names of variables/arguments/parameters/properties
```suggestion
  return grades.reduce((sum, currentGrade) => sum + currentGrade, 0) / grades.length;
```